.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
I guess we've got our wires crossed.
If you ship two separate bundles, one to use for development and one to use in production, the propTypes should be removed from the production bundle altogether. They only take up space and are not used in production anyway.
But if you ship a single bundle that is meant to be used in development as well as in production, there's no point in wrapping the proptypes specification in an `if (__DEV__)` because the bundle will only get larger.